test/external/external_test_onnx_backend.py

```diff
@@ -194,6 +194,11 @@ def supports_device(cls, device: str) -> bool:
 if Device.DEFAULT == 'METAL':
   backend_test.exclude('test_maxpool_2d_same_upper_cpu')
 
+# TODO when ran locally, RuntimeError: OpenCL Error -48
```

Comment:
is this due to metal not supporting float64? does `METAL=1` run?

Reply:
What's weird is that `METAL=1` passes this test...... 
But this is due to the same error when `double` is introduced
For instance, 
`test_cast_DOUBLE_to_FLOAT16_cpu` also raises `OpenCL Error -48`